release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -101,6 +101,44 @@ type of private DNS zone.
 // Monitor whether this info moves to a KB article or remains in official docs.
 // If it moves to KB, permanently remove this entry.
 
+[id="ocp-4-5-machine-api"]
+=== Machine API
+
+[id="ocp-4-5-aws-machinesets-support-spot-instances"]
+==== AWS MachineSets support spot instances
+
+AWS MachineSets now support spot instances. This lets you create a MachineSet
+that deploys machines as spot instances so you can save costs compared to
+on-demand instance prices. You can configure spot instances by adding the
+following line under the `providerSpec` field in the MachineSet YAML file:
+
+[source,yaml]
+----
+providerSpec:
+  spotMarketOptions: 
```

Comment:
I don't think leaving the line trailing empty is valid yaml is it? I've always done it as
```suggestion
  spotMarketOptions: {}
```